packages/vue/src/components/atoms/SfCircleIcon/SfCircleIcon.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-  <SfButton class="sf-circle-icon" :disabled="disabled" v-on="$listeners">
+  <SfButton
+    :aria-label="ariaLabel"
```

Comment:
@martaradziszewska we neet it? when you add on parent component it should work  on it :)

Reply:
it fixes accessibility violations on SfCircleIcon